src/components/quality-page/quality-page.styles.scss

```diff
@@ -0,0 +1,56 @@
+.quality-page-container {
+    background-color: #F8F5F2;
+    height: 290vh;
```

Comment:
Had it in pixels before, but with the 50vw images, it did the same thing. Using vh seemed to be the best. But if i change the images to pixels, that'll help. 
What do you mean by "nest them in the container and add some spacing"?

Reply:
> Had it in pixels before, but with the 50vw images, it did the same thing. Using vh seemed to be the best. But if i change the images to pixels, that'll help.

lets never use vh/vw again 🙌🏻

> What do you mean by "nest them in the container and add some spacing"?

I meant, the container(parent) that holds all the nested components(children) should have its height set by the accumulated height of all its children. rather than set by the screen height(vh).

this is not real code but just to illustrate my point:

```
<container color=red>
    <child height=10>
    <child height=10>
    <child height=10>
</container>
```

the container element would be 30 pixels in height because it grows to the size of its children.
then if you adding 5 padding to the containers top and bottom, it would be 40 pixels high.